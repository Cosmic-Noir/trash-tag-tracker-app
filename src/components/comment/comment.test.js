import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";
import Comment from "./comment";

describe("Comment Component", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDom.render(<Comment />, div);
    ReactDom.unmountComponentAtNode(div);
  });
  it("Renders the UI as expected", () => {
    const tree = renderer.create(<Comment />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
