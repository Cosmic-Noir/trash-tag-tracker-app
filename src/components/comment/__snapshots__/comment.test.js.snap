// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Component Renders the UI as expected 1`] = `[Function]`;
